<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>718</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AsStudio</string>
  </property>
  <widget class="QWidget" name="mainwindow">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>498</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:三极行楷简体-粗;
	font-size:16px;
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;礼物流水&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>20</width>
      <height>691</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="danmaku_frame">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>0</y>
      <width>501</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:三极行楷简体-粗;
	font-size:16px;
	color:rgb(255, 255, 255);
}
.QFrame{
	margin:15px 10px 10px 15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QTextBrowser" name="danmaku">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>23</y>
       <width>461</width>
       <height>681</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color:rgba(255, 255, 255, 150);
color:rgb(0, 0, 0);
font-size:16px;
font-family:黑体;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="control_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:三极行楷简体-粗;
	font-size:16px;
	color:rgb(255, 255, 255);
}
.QFrame&gt;.QFrame:hover{
	background-color:rgba(235, 235, 235, 100);
}
.QFrame{
	margin:15px 10px 10px 15px;
}
QPushButton{
	background-color:rgba(255, 255, 255, 150);
	font-size:14px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="dynamic_frame">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame{
border-width:2px;
border-top-left-radius:2em;
border-top-right-radius:0em;
border-bottom-right-radius:0em;
border-bottom-left-radius:0em;
border-style:solid none none solid; 
border-color:rgb(255, 255, 255);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTextBrowser" name="dynamictextBrowser">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QTextBrowser{
	border-style:none;
	background-color:transparent;
}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'三极行楷简体-粗'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;开启后开播自动发送b站开播动态&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="dynamiclabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">.QLabel{
font-size:20px;
}</string>
         </property>
         <property name="text">
          <string>开播动态</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>160</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="dynamicokButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dynamiccloseButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="weather_frame">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame{
border-width:2px;
border-top-left-radius:0em;
border-top-right-radius:2em;
border-bottom-right-radius:0em;
border-bottom-left-radius:0em;
border-style:solid solid none none; 
border-color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTextBrowser" name="weathertextBrowser">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QTextBrowser{
	border-style:none;
	background-color:transparent;
}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'三极行楷简体-粗'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;弹幕查询格式：&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;#xx(城市)天气&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="weatherokButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="weathercloseButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="weatherlabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">.QLabel{
font-size:20px;
}</string>
         </property>
         <property name="text">
          <string>天气获取</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="puzzle_frame">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame{
border-width:2px;
border-top-left-radius:0em;
border-top-right-radius:0em;
border-bottom-right-radius:0em;
border-bottom-left-radius:2em;
border-style:none none solid solid; 
border-color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="puzzlelabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">.QLabel{
font-size:20px;
}</string>
         </property>
         <property name="text">
          <string>谜语游戏</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="puzzletextBrowser">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>171</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QTextBrowser{
	border-style:none;
	background-color:transparent;
}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'三极行楷简体-粗'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;弹幕游玩格式：&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;①#谜面②#提示&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;③#谜底：xxx&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;④#换题(房管/舰长)&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;⑤#分数(7天未答清零)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_18">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="puzzleokButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="puzzlecloseButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="run_frame">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>290</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame{
border-width:2px;
border-top-left-radius:0em;
border-top-right-radius:0em;
border-bottom-right-radius:2em;
border-bottom-left-radius:0em;
border-style:none solid solid none; 
border-color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="runlabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">.QLabel{
font-size:20px;
}</string>
         </property>
         <property name="text">
          <string>开始运行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="runokButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runcloseButton">
         <property name="styleSheet">
          <string notr="true">*{
	font-size:14px;
}</string>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="runtextBrowser">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QTextBrowser{
	border-style:none;
	background-color:transparent;
}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'三极行楷简体-粗'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16px;&quot;&gt;运行前请确保已经输入机器人账号的cookie&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_20">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>421</width>
       <height>38</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10,0,1">
      <item>
       <widget class="QLabel" name="roomlabel">
        <property name="text">
         <string>房间号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="roomid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLineEdit{
color:rgb(0, 0, 0);
font-size:12px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cookiebutton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>点我输入/修改cookie</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="gift_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>571</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:三极行楷简体-粗;
	font-size:16px;
	color:rgb(255, 255, 255);
}
.QFrame{
	margin:15px 10px 10px 15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QTextBrowser" name="gifts">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>23</y>
       <width>531</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color:rgba(255, 255, 255, 150);
color:rgb(0, 0, 0);
font-size:16px;
font-family:微软雅黑;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>205</y>
      <width>20</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:三极行楷简体-粗;
	font-size:16px;
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;直&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;播&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;弹&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;幕&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
